packages/aws-signer-notation-plugin/aws-signer-notation-plugin.spec

```diff
@@ -0,0 +1,98 @@
+%global goproject github.com/aws
+%global gorepo aws-signer-notation-plugin
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.0.2292
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}aws-signer-notation-plugin
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: AWS Signer plugin for Notation
+License: Apache-2.0
+URL: https://%{goimport}
+Source0: https://%{goimport}/archive/v%{gover}/%{gorepo}-v%{gover}.tar.gz
+Source1: bundled-%{gorepo}-v%{gover}.tar.gz
+
+# The commercial and gov root certificates for AWS Signer.
+Source101: aws-signer-notation-root.crt
+Source102: aws-us-gov-signer-notation-root.crt
+
+BuildRequires: git
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{name}(binaries)
+Requires: %{_cross_os}notation
+Requires: %{_cross_os}ecr-credential-helper
+
+%description
+%{summary}.
+
+%package bin
+Summary: AWS Signer plugin for Notation binaries
+Provides: %{name}(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(fips) or %{name}-fips-bin)
+
+%description bin
+%{summary}.
+
+%package fips-bin
+Summary: AWS Signer plugin for Notation binaries, FIPS edition
+Provides: %{name}(binaries)
+Requires: (%{_cross_os}image-feature(fips) and %{name})
+Conflicts: (%{_cross_os}image-feature(no-fips) or %{name}-bin)
+
+%description fips-bin
+%{summary}.
+
+%prep
+%setup -n %{gorepo}-%{gover} -q
+%setup -T -D -n %{gorepo}-%{gover} -b 1 -q
+
+%build
+%set_cross_go_flags
+
+go build -ldflags "${GOLDFLAGS}" -o notation-com.amazonaws.signer.notation.plugin ./cmd
+gofips build -ldflags "${GOLDFLAGS}" -o fips/notation-com.amazonaws.signer.notation.plugin ./cmd
+
+%install
+install -d %{buildroot}{%{_cross_bindir},%{_cross_fips_bindir},%{_cross_templatedir}}
+
+# Place the binaries where notation expects them.
+install -d %{buildroot}%{_cross_bindir}/notation-plugins
+install -d %{buildroot}%{_cross_bindir}/notation-plugins/plugins
```

Comment:
I saw you have control over this path, it will be best to follow fedora's guidelines and install them in `libexec`.